plugins/node/instrumentation-perf-hooks/test/event_loop_utilization.test.ts

```diff
@@ -102,6 +127,18 @@ describe('nodejs.event_loop.utilization', () => {
       metrics[0].descriptor.description,
       'Event loop utilization'
     );
-    assert.strictEqual(metrics[0].descriptor.unit, '1');
+    assert.strictEqual(
+      metrics[0].descriptor.unit,
+      '1',
+      'expected default unit'
+    );
+    assert.strictEqual(
+      metrics[0].dataPoints.length,
+      1,
+      'expected one data point'
+    );
+    const val = metrics[0].dataPoints[0].value;
+    assert.strictEqual(val > 0, true, `val (${val}) > 0`);
+    assert.strictEqual(val < 1, true, `val (${val}) < 1`);
```

Comment:
> alternately, we can also allow <=1 as a value (because it's a valid value)

Having read the documentation again, I do think the original reason why the tests were flakey was because of the `<`. So I think it doesn't hurt to changed this to `<=`

Reply:
Done in [5a40557](https://github.com/open-telemetry/opentelemetry-js-contrib/pull/1965/commits/5a40557618cee84e27414f0e76e73b95a1347e55) :slightly_smiling_face: